src/shared/slider/slider.css

```diff
@@ -20,7 +21,7 @@
     outline: 0;
     transition: all 0.5s;
     border-radius: 35px;
-    z-index: 1000;
+    z-index: 1;
     border: 0;
     background: rgba(0, 0, 0, 0.5);
     min-width: 35px;
```

Comment:
Suggest changing to _min-width: 25px;_ and _min-height: 25px;_ so it won`t look so massive.